wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+wordpress                                    4.8-apache            fcf3e41b8864        2 years ago         408MB
+```
+
+You can now return to the quickstart home directory.
+
+### Step 2: Creating the Kubernetes Resources
+
+Making Wordpress run on Kubernetes is easy. The quickstart repository you have
+cloned above includes the YAML definitions of Kubernetes objects. We'll use
+a Helm chart for this:
+
+```
+$ tree wordpress/resources/charts/wordpress/
+wordpress/resources/charts/wordpress/
+├── Chart.yaml
+├── templates
+│   ├── _helpers.tpl
+│   ├── mysql-deployment.yaml
+│   ├── secret.yaml
+│   └── wordpress-deployment.yaml
+└── values.yaml
+
+```
+
+The values.yaml specifies several important values including
+  - Using OpenEBS for storage
+  - Size of the Persistent Storage for Wordpress and MySQL
+  - Password for MySQL db.
+You are welcome to modify this file and you can use --set and --values in the tele build process to replace these values. In this tutorial, we are packaging a single Helm chart but it is possible to have several of them packaged into a single Cluster Image.
+
+
+### Step 4: Building the Cluster Image
+Let's build the cluster image:
+
+```bsh
+$ tele build -o wordpress.tar wordpress/resources/app.yaml
+Mon Apr 27 00:49:02 UTC Building cluster image wordpress 0.0.1
+Mon Apr 27 00:49:02 UTC Selecting base image version
+        Will use base image version 7.0.1
+Mon Apr 27 00:49:02 UTC Downloading dependencies from s3://hub.gravitational.io
+        Still downloading dependencies from s3://hub.gravitational.io (10 seconds elapsed)
+        Still downloading dependencies from s3://hub.gravitational.io (20 seconds elapsed)
+Mon Apr 27 00:50:31 UTC Embedding application container images
+        Pulling remote image quay.io/gravitational/debian-tall:0.0.1
+        Pulling remote image quay.io/gravitational/debian-tall:stretch
+        Pulling remote image quay.io/gravitational/provisioner:ci.82
+        Pulling remote image quay.io/gravitational/debian-tall:buster
+        Using local image mysql:5.6
+        Using local image wordpress:4.8-apache
+        Vendored image gravitational/debian-tall:0.0.1
+        Vendored image gravitational/debian-tall:stretch
+        Vendored image gravitational/debian-tall:buster
+        Vendored image gravitational/provisioner:ci.82
+        Vendored image mysql:5.6
+        Vendored image wordpress:4.8-apache
+Mon Apr 27 00:51:21 UTC Creating application
+        Still creating application (10 seconds elapsed)
+Mon Apr 27 00:51:36 UTC Generating the cluster image
+Mon Apr 27 00:51:44 UTC Saving the image as wordpress.tar
+        Still saving the image as wordpress.tar (10 seconds elapsed)
+Mon Apr 27 00:52:02 UTC Build finished in 3 minutes 
+```
+
+Let's review what just happened. `tele build` did the following:
+
+* Downloaded Kubernetes binaries and Gravity tooling from `s3://hub.gravitational.io`.
```

Comment:
Imo the s3 storage is implementation-detail. Not sure why this is being displayed as a progress message.

Reply:
Changed to just Kubernetes binaries and Gravity tooling.  Also put in Roman's edit.